x-pack/plugins/maps/public/classes/tooltips/tooltip_property.ts

```diff
@@ -59,10 +59,10 @@ export class TooltipProperty implements ITooltipProperty {
   }
 
   getHtmlDisplayValue(): string {
-    return _.escape(this._rawValue);
+    return _.escape(Array.isArray(this._rawValue) ? this._rawValue.join() : this._rawValue);
```

Comment:
Same comment as above @nreese: 

> what would be the desired behavior here for array values? Should this join as is, with a , separator, just show the first value (existing impl), or something else?

Reply:
I think this is fine here since we have no knowledge of the raw value type at this point.